LeadKit/LeadKit/Protocols/CellsControllerProtocol.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CellsControllerProtocol.swift
+//  LeadKit
+//
+//  Created by Ivan Smolin on 08/06/16.
+//  Copyright © 2016 Touch Instinct. All rights reserved.
+//
+
+import Foundation
+
+/**
+ *  protocol which ensures that specific type can configure cell and return cell attributes for specific index path
+ */
+public protocol CellsControllerProtocol {
+    associatedtype CellType
```

Comment:
Should not we use `typealias CellType = ...` in classes, which conforms this protocol?